### PR TITLE
Jeff

### PR DESCRIPTION
**Pull request by jeffbum**

### CODE REVIEW

File: pitches/jeff-bumgardner-project-brewlive.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Project Pitch Name (Jeff Bumgardner)
```
Comment (dummied) [inline pitches/jeff-bumgardner-project-brewlive.md]: Is your project called "Project Pitch Name"?

File: pitches/jeff-bumgardner-project-brewlive.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+Brew Live (or Live Brew, depends on how many you've had.)
+
+## Elevator
+Home brewing can be an expensive hobby when someone is doing it on there own. Why isn't there a place that not only allows to you to meet up with fellow home brewers, but also allows to maximize the ingredients, tools and knowledge you possess? Luckily, Brew Live is in the early stages of being built. Muahahaha!!
```
Comment (dummied) [inline pitches/jeff-bumgardner-project-brewlive.md]: on _their_ own

File: pitches/jeff-bumgardner-project-brewlive.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Audience
+
+This application will target those who would like to brew their own beer instead of buying pre-made beer. That would be the vast majority of the population. Some sub-populations could be:
```
Comment (dummied) [inline pitches/jeff-bumgardner-project-brewlive.md]: I suspect you overestimate the appeal of home brewing, but that's a fine niche for this.

File: your-name-project-name.md [hunk 4]
```diff
@@ -8,6 +8,7 @@ A one or two sentence synopsis of what your idea is.
 
 Who will use this product/application/thing.
 
+
```
Comment (dummied) [inline your-name-project-name.md]: Why'd you add an empty line here?

Comment (jeffbum) [inline your-name-project-name.md]: testing the branch aspect of pushing. Didn't use it in either of the projects.

File: pitches/jeff-bumgardner-project-brewlive.md (hunk 3 above)
Comment (jeffbum) [inline pitches/jeff-bumgardner-project-brewlive.md]: I meant that the vast majority of the population that would use this application would be home brewers. Not that there are a lot of HB's.

**Pull request closed without merging**